README.md

```diff
@@ -161,17 +161,25 @@ git clone https://github.com/ggerganov/llama.cpp
 cd llama.cpp
 ```
 
-### Build
+### Build with Linux/ Mac Using Make
 
-In order to build llama.cpp you have three different options.
+These commands are specific to Ubuntu linux but OS specific varients are just a google away given this handy dependency list.  Also, if you're using your windows gaming machine, some users have reported great success in using [WSL2](https://github.com/ggerganov/llama.cpp/issues/103#issuecomment-1470440202) to install Ubuntu within Windows and following the linux build instructions to run this project.
```

Comment:
I can't speak for which OS the author had in mind, but these commands work on Debian and any Debian based distros, including Ubuntu, the infamous Kali, and even Raspberry Pi OS (previously Raspbian).

Also, while WSL2 works great for building this project in a Windows environment (I didn't even have to edit any files like the guy in that link), it's not as easy to get BLAS support working and the speed difference alone is probably worth building a native Windows app (at least for anyone with a modern video card).

Reply:
Thanks, I was interested in the higher parameter models and found that I needed the below snippet to use more of my system's resources.  

```
[wsl2]
memory=51GB
processors=10
swap=4GB
```

I was struggling between keeping the content concise and as minimal as possible so balanced here.  I'm open to changing things of course.